### DISCUSSION TITLE
Unrecognized Language Error CoPilot Chat

### DISCUSSION THREAD

**Discussion by blrcoderbro** [community/community#72706] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Unrecognized Language Error While Using Fix in VS Code

https://ibb.co/2s4wNRr

**Comment by Akash1134** [comment]
Hi @codedivinetech , we're happy you're here! You are more likely to get a useful response if you are explicit about issue providing more details. Please share the version of Copilot and VS Code you're working with along with the set of prompts that you think are returning unrecognized error in Copilot Chat. 

Thanks!

**Comment by Ricordy** [comment]
Same issue here! 
<img width="707" alt="image" src="https://github.com/community/community/assets/42589197/655c5efa-d427-4b86-86a7-7571bc36048d">
Im using the latest copilot and ubuntu versions as well as the vs code, since i installed this machine 5 days ago

**Comment by blrcoderbro** [comment]
@Akash1134 

Image : https://ibb.co/yhDmrP3

![image](https://github.com/community/community/assets/74067051/c580a040-fd38-4b49-8df5-790a2a063f43)

Some Debug Information For You

Version: 1.83.1 (system setup)
Commit: f1b07bd25dfad64b0167beb15359ae573aecd2cc
Date: 2023-10-10T23:48:05.904Z
Electron: 25.8.4
ElectronBuildId: 24154031
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22621

## Copilot

- Version: 1.130.0
- Build: prod
- Editor: vscode/1.83.1

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Custom Certificates: disabled
- Send Restricted Telemetry: enabled

## Node setup

- Number of root certificates: 140
- Operating system: Windows_NT
- Operating system version: 10.0.22621
- Operating system architecture: x64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Network Configuration

- Proxy host: n/a
- Proxy port: n/a
- Proxy auth: n/a
- Kerberos SPN: n/a
- Reject unauthorized: disabled

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 - 

## VS Code Configuration

- HTTP proxy: 
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Extensions

- Is `win-ca` installed?: false
- Is `mac-ca` installed?: false

**Reply by blrcoderbro** [reply]
@Akash1134 Please Check this Issue and Help Us

**Reply by blrcoderbro** [reply, marked as answer]
Issue is Resolved when i switched to Pre-Release Version it upgraded me to v0.10.0

![image](https://github.com/community/community/assets/74067051/4249c461-ecb1-4b9a-9b83-615345c20baf)

**Comment by lecajer** [comment]
Switching to Pre-Release keeps version 8 on my VS code with `Unrecognized Language: php`

Kind of funny by the way :
![image](https://github.com/community/community/assets/2142505/27acdf4b-5534-4906-8144-7209fab151b0)
